swarm_copy/app/database/sql_schemas.py

```diff
@@ -0,0 +1,48 @@
+"""Schemas for the chatbot."""
+
+import datetime
+import uuid
+
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import declarative_base, relationship
+
+Base = declarative_base()
+
+
+def uuid_to_str() -> str:
+    """Turn a uuid into a string."""
+    return uuid.uuid4().hex
+
+
+class Threads(Base):
```

Comment:
As discussed on slack, I would suggest we switch to this new recommended `Mapped` and `mapped_column` syntax

https://docs.sqlalchemy.org/en/20/orm/declarative_tables.html#declarative-table-with-mapped-column

Reply:
<img width="687" alt="Screenshot 2024-10-31 at 11 46 31" src="https://github.com/user-attachments/assets/bb252086-da88-4071-9492-3ab00fcef682">

Reply:
fixed in e60ad53